File: Solution/CSharp/58.LengthofLastWord.cs
public class Solution {
    public int LengthOfLastWord(string s) {
        int sum = 0;
        bool isWorld = false;
        for (int i = s.Length - 1; i >= 0; i--) {
            if (!isWorld && s[i] != ' ') {
                isWorld = true;
            }
            if (isWorld == true && s[i] != ' ') {
                sum++;
            } else if (isWorld == true && s[i] == ' ') {
                break;
            }
        }
        return sum;
    }
}

File: Solution/Python/58. Length of Last Word.py
class Solution:
    def lengthOfLastWord(self, s):
        """
        :type s: str
        :rtype: int
        """
        return len(s.rstrip(" ").split(" ")[-1])


s = Solution()
#string = "Hello World"
string = "a "
res = s.lengthOfLastWord(string)
print(res)
